### PR TITLE
Add prettier and lint, issue #10 

### PR DESCRIPTION
**Pull request by catorda**

### CODE REVIEW

**Comment by catorda** [comment]
This is to fix issue #10

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (vvscode) [inline .jshintrc]: don't think we need that tool
it's better to use `eslint`

File: .travis.yml [hunk 2]
```diff
@@ -5,4 +5,6 @@ cache:
   directories:
     - node_modules
 script:
-  - npm test
\ No newline at end of file
+  - npm test
+  - npm run prettier
```
Comment (vvscode) [inline .travis.yml]: what for do we need prettier here? results won't be saved in repo

I was talking about using `precommit` hook for formatting code

File: .jshintrc (hunk 1 above)
Comment (catorda) [reply .jshintrc]: Sorry, I'll switch it to eslint.

File: .travis.yml (hunk 2 above)
Comment (catorda) [reply .travis.yml]: Oh got it. I'll change it to a precommit hook that will change the formatting before committing to the repo.

**Comment by catorda** [comment]
For setting up the git precommit hook, I added a 'setup' script in the package.json. So first contributors would run, 'npm run setup', this adds the git commit prehook by setting the git config for it. Let me know if you want it done a different way. Or if you want me to add a 'how to' to a contributing doc.

File: hooks/pre-commit [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```
Comment (vvscode) [inline hooks/pre-commit]: this file won't be applied automatically ( hooks stored in `.git/hooks` so it's our of git repo) - use `husky` package and `lint-staged` for running on staged files

**Review by vvscode — CHANGES_REQUESTED** [review]
Manual hooks are not coold

husky / lint-staged packages from npm

**Pull request merged**